### PR TITLE
Feature/points

### PR DESCRIPTION
**Pull request by patryklomza**
This is a starting point for our Points system.

Check updated README for a brief endpoints description.

I will add an endpoint calculating total student points in a later PR. 

Waiting for your questions, and suggestions on what to improve!

### CODE REVIEW

**Comment by visse0001** [comment]
Could you push 'makemigrations'?

Nice README description. 
And the tests! Too bad when I run them I get " no tests ran in 0.09s" :thinking: 
After hitting endpoint /api/users/points/?studentId=2 I have: 
```
"AssertionError at /api/users/points/
'PointResource' should either include a `serializer_class` attribute, or override the `get_serializer_class()` method."
```

**Comment by visse0001** [comment]
Could you move `pytest.ini` and tests to root dir? ;)

**Comment by patryklomza** [comment]
> Could you move `pytest.ini` and tests to root dir? ;)

✅ done

**Comment by patryklomza** [comment]
> Could you push 'makemigrations'?
> 
> Nice README description.
> And the tests! Too bad when I run them I get " no tests ran in 0.09s" 🤔
> After hitting endpoint /api/users/points/?studentId=2 I have:
> 
> ```
> "AssertionError at /api/users/points/
> 'PointResource' should either include a `serializer_class` attribute, or override the `get_serializer_class()` method."
> ```

✅ done

File: users/resources.py [hunk 1]
```diff
@@ -109,3 +110,35 @@ def patch(self, request, pk):
     def delete(self, request, pk):
         Task.objects.filter(pk=pk).delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class PointResource(generics.GenericAPIView):
```
Comment (visse0001) [inline users/resources.py:115]: Looks good. Could you add the serializer_class attribute here? and let’s merge this :)

File: README.md [hunk 2]
```diff
@@ -167,3 +167,35 @@ Go to
 http://localhost:8000/api/users/tasks/  
 http://localhost:8000/api/users/tasks/1  
 
+## Points endpoints  
+There are currently several enpoints responsible for managing student points available:
+
+1. `api/users/points/` accesible with a `GET` request and followed by a query string pointing to a student primary key.  
+For example: `api/users/points/?studentId=2/` should return Point instances assigned to the student with `pk=2`
```
Comment (visse0001) [inline README.md]: `api/users/points/?studentId=2/` this trailing slash is redundant and couses ValueError :)

**Review by visse0001 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (patryklomza) [reply README.md]: ✅ done

File: users/resources.py (hunk 1 above)
Comment (patryklomza) [reply users/resources.py:115]: ✅ done

**Pull request merged**